docs/en/operations/utilities/clickhouse-benchmark.md

```diff
@@ -28,35 +28,35 @@ $ clickhouse-benchmark [keys] <<< "single query"
 If you want to send a set of queries, create a text file and place each query on the individual string in this file. For example:
 
 ``` sql
-SELECT * FROM system.numbers LIMIT 10000000
-SELECT 1
+SELECT * FROM system.numbers LIMIT 10000000;
+SELECT 1;
 ```
 
-Then pass this file to a standard input of `clickhouse-benchmark`.
+Then pass this file to a standard input of `clickhouse-benchmark`:
 
 ``` bash
-clickhouse-benchmark [keys] < queries_file
+clickhouse-benchmark [keys] < queries_file;
 ```
 
 ## Keys {#clickhouse-benchmark-keys}
 
--   `--query=WORD` - Query to execute. If this parameter is not passed clickhouse-benchmark will read queries from standard input. 
+-   `--query=WORD` — Query to execute. If this parameter is not passed, `clickhouse-benchmark` will read queries from standard input. 
```

Comment:
I think `--query=QUERY` will be better, cause usually query is not a single word. E.g. `clickhouse-benchmark --query "SELECT 1"`. 

Also `-h HOST`